op-e2e/actions/upgrades/interop_fork_test.go

```diff
@@ -0,0 +1,138 @@
+package upgrades
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/interop/dsl"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/lmittmann/w3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInteropUpgradeTransactions(gt *testing.T) {
+	t := helpers.NewDefaultTesting(gt)
+
+	// Run Isthmus until the interop upgrade
+	is := dsl.SetupInterop(t, dsl.SetInteropOffset(uint64(15)))
+	actors := is.CreateActors()
+	actors.PrepareChainState(t)
+
+	////////////////////////////
+	// Pre-upgrade Block Production
+	////////////////////////////
+
+	fmt.Println("Pre-upgrade Block Production")
```

Comment:
if you want to log in tests, better use `t.Log` (or `gt.Log` here?)

Reply:
Removed println in c6fd3b743583a4bdbc626c27c8e5b1a8af210d24.